### ISSUE TITLE
[Feature] sync last server using a toggle in the config

### ISSUE THREAD

**Issue by jnijland** [ProxioDev/RedisBungee#84]
**Describe the bug**
TL;DR Not sure if this is something that is supposed to be working or not, maybe it's a new feature. 

When on a multi proxy setup, with the force_default_server option set to false, the player is not sent to the latest server it was joined to. It does work when the user joins the same proxy, but not when switching proxies. 

For example; player is on proxy a on server minigames_1. When relogging to proxy a player gets send to minigames_1, however when joining via proxy b it gets send to the default server of proxy b.

**To Reproduce**
1. Have a multi proxy setup with multiple Bungeecord/Waterfall/Flamecord proxies; 
2. Set reconnect to enable in ```modules.yml``` and ```force_default_server``` to false in the ```config.yml```;
3. Have multiple backend servers behind the proxies; 
4. Join one of the proxies and go to a non-default backend server, disconnect;
5. Join a different proxy, you will be sent to the default backend server;

**Expected behavior**
The player should be sent to the last server it was on 

**Redis version? it should be at least 6 and above.**
6.0.16

**Bungeecord version or (the bungee fork name eg: waterfall) and your plugins**
 Flamecord 1.5.9 but also tested on Waterfall 1.20 build 536.

**Comment by ham1255** [comment]
hmm i don't think this a thing in redisbungee, but i could include in future update, as toggle in config in-case you handle it differently

**Comment by jnijland** [comment]
Would be awesome @ham1255! It might be an edge case since most people want players to always join in a lobby, but I'm sure there are more server owners out here who would like to use a multi proxy setup using RedisBungee with players joining their last server.
